### PR TITLE
[DOCS] Update index.rst

### PR DESCRIPTION
**Pull request by abegong**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [ENHANCEMENT], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Fills in gaps in Getting Started
- Makes bash commands, code blocks, JSON objects, screen shots, etc. consistent through the tutorial
- Applies styling to intro.rst

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

Thank you for submitting!

### CODE REVIEW

**Comment by abegong** [comment]
This is ready to go!

**Comment by abegong** [comment]
@jcampbell Any concerns about these changes? I'd love to get them merged before all the good people come back from the weekend.

File: docs/tutorials/getting_started/initialize_a_data_context.rst [hunk 1]
```diff
@@ -87,33 +99,22 @@ You should see this:
...
-
-First, Great Expectations will create a new directory:
-
-.. code-block::
-
         great_expectations
-        |-- expectations
         |-- great_expectations.yml
-        |-- checkpoints
+        |-- expectations
```
Comment (Aylr) [inline docs/tutorials/getting_started/initialize_a_data_context.rst:108]: The checkpoints directory is missing.

**Review by Aylr — APPROVED** [review]
FF to merge this after 1 small change is made.

**Comment by abegong** [comment]
Good catch, @Aylr .

**Pull request merged**